README.md

```diff
@@ -36,28 +33,46 @@ In this example we have a `NewsHolder` page which is the root of our news sectio
 ```php
 <?php
 
+namespace MyModule\PageTypes;
+
 use Page;
 use SilverStripe\Lumberjack\Model\Lumberjack;
 
-class NewsHolder extends Page
+class StaffHolder extends Page
 {
     private static $extensions = [
         Lumberjack::class,
     ];
 
     private static $allowed_children = [
-        'NewsArticle',
-        'NewsPage',
+        StaffArticle::class,
```

Comment:
Hmm, on second thought I'm not so sure this analogy fits with the 3 page types (I was only thinking of 2 when we talked about it - holder and page), article doesn't sound quite right.

The confusion in the crossover with [cwp/cwp](https://github.com/silverstripe/cwp/tree/master/src/PageTypes) is confusing I still agree though. But perhaps it's not particularly worth worrying about in this case, unless we can think up something else that fits better.

Perhaps the addition of the namespace alone is enough to remove that confusion?

Reply:
That's a fair point. I will revert it to use the news examples, and you are right, the namespace should be enough to remove the confusion.